### ISSUE TITLE
Include copyright notices

### ISSUE THREAD

**Issue by jpadilla** [uw-labs/lichen#23]
Would it make sense to also include any NOTICE found in packages? This is something that `go-licenses` can do https://github.com/google/go-licenses/blob/5b654af5dcd3ef8090baaceae6009c20d75a87e8/save.go#L109

**Comment by nick-jones** [comment]
I'm guessing this is with some auditing type requirement in mind? So essentially just identify where these copyright notices are included, but not actually run any specific checks over them?

**Comment by jpadilla** [comment]
Correct.  I think `lichen` can detect what type of license a library has and fulfill its requirements, e.g. copy license, copyright notice, source code, etc.

Ideal state for json output: it already does licenses just fine, it could include contents of any copyright notices, and could potentially point to needing to copy entire source directory for a dependency.
